app/services/chonker.rb

```diff
@@ -24,8 +35,6 @@ def chunks
     end
   end
 
-  private
```

Comment:
Did you intend to make `chunking_method` public?

Reply:
Nope. I moved `private` up a little to _also_ make `#chunks` private. This remains private.

Reply:
Oops!